### ISSUE TITLE
completed-docs should have an option to check every method/function overload

### ISSUE THREAD

**Issue by SerhiiBilyk** [palantir/tslint#4416]
### Feature request for `completed-docs` rule
Is it possible to enforce (add option if does not exist) jsDoc comment for function/method overloading?

**Comment by JoshuaKGoldberg** [comment]
@SerhiiBilyk sorry, what do you mean by "overloading"? Could you post a code snippet or two?

**Comment by SerhiiBilyk** [comment]
@JoshuaKGoldberg 
```typescript
class Foo {

    /*
     * Enforce jsDoc comment here
     */
    emit(event: string, data: number) => void;

    /*
     * Enforce jsDoc comment  here
     */
    emit(event:string, data: string) => void;
 
    emit(event:string, data:any) {
        // ....... some code
    }
}
```

**Comment by 0815fox** [comment]
Did you do something on this? Because since my update from 5.12.x to 5.13.0 today, i get new reports for exactly that and I definitly do not want to write a separate comment for each overload ;-) If so: is there a not yet documented option to turn this off?

**Comment by adidahiya** [comment]
hmm, that could have happened as a result of #3557. @JoshuaKGoldberg could you take a look?

**Comment by JoshuaKGoldberg** [comment]
Yes, the behavior is now to require one on all overloads.

```json
{
  "rules": {
    "completed-docs": [
      true,
      {
        "methods": true
      }
    ]
  }
}
```

```ts
class Foo {
    // completed-docs: Documentation must exist for methods
    emit(event: string, data: number): void;

    // completed-docs: Documentation must exist for methods
    emit(event:string, data: string): void;
 
    /**
     * Exists in one place
     */
    emit(event:string, data:any) {
        // ....... some code
    }
}
```

**Comment by 0815fox** [comment]
Thumbs up for configurable. Suggestion:

```
"overloads": false
```
